spec/requests/api/v1/conversations/index_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Conversation', type: :request do
+  let(:user) { create(:user) }
+  let(:receivers) { create_list(:user, 4) }
+  let(:topic) { create(:topic) }
+  let(:conversations_response) { json['conversations'] }
+  let!(:conversations) { ConversationService::Creator.new(sender: user, receivers: receivers, topic: topic).call }
+
+  describe 'GET /api/v1/conversations' do
+    subject { get api_v1_conversations_path, headers: auth_headers, as: :json }
+
+    context 'when user is not logged in' do
+      let(:auth_headers) { nil }
+
+      before { subject }
+
+      it 'returns unauthorized status' do
+        expect(response).to have_http_status(:unauthorized)
+      end
+
+      it 'returns an error message' do
+        expect(errors.first).to eq I18n.t('api.errors.user_not_logged')
```

Comment:
I prefer to test with the plain message here, so you also can check when the message changes, you can detect it